packages/core/feature-flags/README.mdx

```diff
@@ -20,56 +20,53 @@ One example of invalid use cases are settings used during the `setup` lifecycle
 
-## Registering a feature flag
+## Key concepts
 
-> [!IMPORTANT]
-> At the moment, we follow a manual process to manage our feature flags. Refer to [this repo](https://github.com/elastic/kibana-feature-flags) to learn more about our current internal process.
-> Our goal is to achieve the _gitops_ approach detailed below. But, at the moment, it's not available, and you can skip it if you want.
+### Feature Flag
 
-Kibana follows a _gitops_ approach when managing feature flags. To declare a feature flag, add your flags definitions in
-your plugin's `server/index.ts` file:
+A config key that defines a set of [variations](#variations) that will be resolved at runtime when the app calls [the evaluation APIs](https://docs.elastic.dev/kibana-dev-docs/tutorials/feature-flags-service#evaluating-feature-flags). The variation is decided at runtime based on static binary state (ON -> `variationA` vs. OFF -> `variationB`), or via [evaluation/segmentation rules](#evaluationsegmentation-rules).
 
-```typescript
-// <plugin>/server/index.ts
-import type { FeatureFlagDefinitions } from '@kbn/core-feature-flags-server';
-import type { PluginInitializerContext } from '@kbn/core-plugins-server';
-
-export const featureFlags: FeatureFlagDefinitions = [
-  {
-    key: 'myPlugin.myCoolFeature',
-    name: 'My cool feature',
-    description: 'Enables the cool feature to auto-hide the navigation bar',
-    tags: ['my-plugin', 'my-service', 'ui'],
-    variationType: 'boolean',
-    variations: [
-      {
-        name: 'On',
-        description: 'Auto-hides the bar',
-        value: true,
-      },
-      {
-        name: 'Off',
-        description: 'Static always-on',
-        value: false,
-      },
-    ],
-  },
-  {...},
-];
-
-export async function plugin(initializerContext: PluginInitializerContext) {
-  const { FeatureFlagsExamplePlugin } = await import('./plugin');
-  return new FeatureFlagsExamplePlugin(initializerContext);
-}
-```
+### Variations
+
+All the potential values that a feature flag can return based on the [evaluation rules](#evaluationsegmentation-rules). A feature flag should define at least 2 variations: the ON and the OFF states. The typical use case sets the OFF state to match the `fallback` value provided to [the evaluation APIs](https://docs.elastic.dev/kibana-dev-docs/tutorials/feature-flags-service#evaluating-feature-flags), although it's not a hard requirement and Kibana contributors might require special configurations.
+
+### Evaluation/Segmentation rules
+
+Set of rules used to evaluate the variation to resolve, based on the [evaluation context](#evaluation-context) provided by the app.
+
+The rules can vary from a percentage rollout per evaluation context, or more complex IF...THEN filters and clauses.
+
+Refer to the [Evaluation Context guide's examples](./evaluation-context.md#examples) for some typical scenarios.
```

Comment:
```suggestion
Refer to the [Evaluation Context guide's examples](https://github.com/elastic/kibana-feature-flags/blob/main/docs/evaluation-context.md#examples) for some typical scenarios.
```

---

I'll take this opportunity to ask. Are we ok to post these many private links in the public repo? This docs seem very internal so it feels fine to me. But just double checking.